packages/design-system/src/components/Link/Link.tsx

```diff
@@ -60,13 +58,13 @@ const Link = React.forwardRef(
 					defaultValue: 'This link is disabled',
 				});
 			}
-			if (isExternal && isBlank && title) {
+			if (isBlank && title) {
```

Comment:
please in that case could you also add a security fix ?
`rel="noopener noreferrer"`

Reply:
We already have it as long as the target is set to _blank
```
rel={isBlank ? 'noopener noreferrer' : undefined}
``` 
Or maybe i misunderstood your point

Reply:
oh didn't see it thanks !